nwHacks first release

Ciuffi opened the pull request:
ITS READY!

Thank you to everyone who worked really hard to get this finished on time.

Here is the preview: https://nwhacks-2019.firebaseapp.com/

ianmah commented:
Some notes but nothing blocking release:

- I think we should only have the team animation once - after you scroll up it shouldn't play again if  you scroll down. 
- I think the footer elements like (c) NwHacks 2020 etc should go below the team section. The team section is big enough that it feels like page content not footer content to me

Ciuffi commented:
hmm I agree, I'll bring up these changes with design.

Originally it didn't reset, but everyone at roundtable wanted a reset .-.

jvssicawu commented on components/NavBar.vue:
```diff
@@ -15,12 +16,11 @@
-            <a v-scroll-to="'#about'" href="#" class="navbar-item">About</a>
+            <a v-scroll-to="'#events'" href="#" class="navbar-item">About</a>
             <a v-if="faq" v-scroll-to="'#faq'" href="#" class="navbar-item">FAQ</a>
             <a v-scroll-to="'#sponza'" href="#" class="navbar-item">Sponsors</a>
```
can we link to last year's nwHacks please, i personally found it fun to look at past websites

Ciuffi replied:
I don't think last years website is up?

Ciuffi replied:
oh it is

Ciuffi replied:
@luceliu and this

Ciuffi replied:
@luceliu acutally I'll add the nwHacks link, you do the logo.

jvssicawu commented on components/NavBar.vue:
```diff
@@ -15,12 +16,11 @@
-          <a v-scroll-to="'#navbar'" href="#">
+          <a v-scroll-to="'#intro'" href="#" target="_blank" rel="noopener">
             <img
               id="navbar-logo"
```
the navbar is still using the nwPlus logo when we should have the _nwHacks_ logo!! pls check the text carefully, i understand it's super small so it might have been overlooked

Ciuffi replied:
noted

Ciuffi replied:
@luceliu can you handle this? You can just commit it straight to dev lol

Ciuffi replied:
nvm i got it

jvssicawu commented on components/NavBar.vue:
```diff
@@ -41,6 +41,28 @@ export default {
+    window.addEventListener('scroll', this.handleScroll())
+  },
+  methods: {
+    handleScroll() {
```
not sure if this is messing up mobile scroll because I can't scroll properly on iphone 7+ the website gets jammed and stuck

jvssicawu replied:
it scrolls properly if i hold and move my finger up/down with the screen, but if i do a quick scroll where i flick my finger and let the screen scroll on its own, it gets stuck? not sure if its a me-only thing

luceliu replied:
scrolling is fine for me on iPhone 6s (Safari).. only slight issue is that it's a bit choppy when you scroll back up to the hero image at top

Ciuffi replied:
Scrolling should be fixed. Could you check it out?

jvssicawu replied:
it's beautiful thank u 
A TRU HERO

jvssicawu commented:
> Some notes but nothing blocking release:
> 
> * I think we should only have the team animation once - after you scroll up it shouldn't play again if  you scroll down.
> * I think the footer elements like (c) NwHacks 2020 etc should go below the team section. The team section is big enough that it feels like page content not footer content to me

Yeah referring the the footer element, i think we can just move the whole social media icons + links down to replace the "made with love by nwPlus." people brought up that it was redundant to have both "organized by nwPlus" and "made with love by nwPlus"

[![Image from Gyazo](https://i.gyazo.com/9164614fd41eedfc87c7848444109133.png)](https://gyazo.com/9164614fd41eedfc87c7848444109133)

jvssicawu commented:
also did you guys remove the linkedin hover state? just wanted to know if yall changed your minds or smth cuz i dont see it

Ciuffi commented:
> also did you guys remove the linkedin hover state? just wanted to know if yall changed your minds or smth cuz i dont see it

its broken on prod.. not totally sure why yet. @AustinKobayashi is looking into it.

Ciuffi commented:
> > Some notes but nothing blocking release:
> > 
> > * I think we should only have the team animation once - after you scroll up it shouldn't play again if  you scroll down.
> > * I think the footer elements like (c) NwHacks 2020 etc should go below the team section. The team section is big enough that it feels like page content not footer content to me
> 
> Yeah referring the the footer element, i think we can just move the whole social media icons + links down to replace the "made with love by nwPlus." people brought up that it was redundant to have both "organized by nwPlus" and "made with love by nwPlus"
> 
> [![Image from Gyazo](https://camo.githubusercontent.com/8bbef5eb6a32f3538c5f3ffc267d0977f001fa31/68747470733a2f2f692e6779617a6f2e636f6d2f39313634363134666434316565646663383763373834383434343130393133332e706e67)](https://gyazo.com/9164614fd41eedfc87c7848444109133)

Just the links/logos and not the banner? That might take a bit, they're tied together.

jvssicawu commented:
> > > Some notes but nothing blocking release:
> > > 
> > > * I think we should only have the team animation once - after you scroll up it shouldn't play again if  you scroll down.
> > > * I think the footer elements like (c) NwHacks 2020 etc should go below the team section. The team section is big enough that it feels like page content not footer content to me
> > 
> > 
> > Yeah referring the the footer element, i think we can just move the whole social media icons + links down to replace the "made with love by nwPlus." people brought up that it was redundant to have both "organized by nwPlus" and "made with love by nwPlus"
> > [![Image from Gyazo](https://camo.githubusercontent.com/8bbef5eb6a32f3538c5f3ffc267d0977f001fa31/68747470733a2f2f692e6779617a6f2e636f6d2f39313634363134666434316565646663383763373834383434343130393133332e706e67)](https://gyazo.com/9164614fd41eedfc87c7848444109133)
> 
> Just the links/logos and not the banner? That might take a bit, they're tied together.

Yeah just the links/logos please! I think there's value in keeping the other stuff on top of the team section just for better visibility

Ciuffi commented:
@jvssicawu Okay I added nwhacks to the navbar, and moved the footer. In the future, this is defs an early type of request, it takes a lot of work to do it last minute .-.

Ciuffi commented:
@luceliu @jvssicawu I think I'll push deploying to tomorrow, so we can fix all this last minute stuff.

@jvssicawu what do you think about the team section reset? is it good or nah

jvssicawu commented:
> @jvssicawu Okay I added nwhacks to the navbar, and moved the footer. In the future, this is defs an early type of request, it takes a lot of work to do it last minute .-.

much appreciate <3

jvssicawu commented:
> would like the bottom of the page to have a bit more of a margin or padding so that the text isnt touching the bottom of the page but ill approve for everything else

bringing this up because im sure the team will bring it up too if i dont (sorry i rly didnt want to nitpick)
